bql/semantic/expression.go

```diff
@@ -222,6 +221,9 @@ func (e *comparisonForNodeLiteral) Evaluate(r table.Row) (bool, error) {
 	if err != nil {
 		return false, fmt.Errorf("in comparisonForNodeLiteral.Evaluate got error: %v", err)
 	}
+	if eL.N == nil {
+		return false, fmt.Errorf("only a node binding should be compared with a node literal, but got %v instead", eL.String())
```

Comment:
This could also have a mix of correct and incorrect rows at runtime so I think we should log the error message and return false with no error.

Point of notice: maybe we should add these swallowed errors to the tracer and if something do go wrong the tracer could signal the swallowed bits or the client could request the trace as well, and check these kind of errors.

Reply:
I see. I will let it to be added to the tracer then and remove this error from here, for now.